controllers/controller.js

```diff
@@ -78,6 +78,17 @@ const getUsers = (req, res, next) =>{
     })
 }
 
+const deleteCommentsById = (req, res, next) => {
```

Comment:
This function name could be clearer, we're able to delete one comment at a time so `deleteCommentById` (which is singular) would be a better description of the behaviour.

Reply:
Thanks for pointing this out, i'll make the change before merge!